CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ##### Enhancements
 
+* Replace git-based MasterSource with CDN-based TrunkSource
```

Comment:
nit wrap MasterSource and TrunkSource with \`\` for fomatting.

Also need to add two empty spaces at the end of this line for formatting.